fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -874,6 +877,14 @@ public long getMaxWriteLimitBytes() {
             return maxWriteLimitBytes;
         }
 
+        /**
+         * Exit with exception if this limit is exceeded (checked after each non-final iteration).
```

Comment:
Is "iteration" a term we've used before in the docs here? I wonder if "transaction" might be a better term (i.e., if that's something a user might already know about).

Reply:
"transaction" is much better. Changed.